chains/goerli.json

```diff
@@ -0,0 +1,145 @@
+{
+  "name": "goerli",
+  "chainId": 6284,
+  "networkId": 6284,
+  "comment": "Cross-client PoA test network",
+  "url": "https://github.com/goerli/testnet",
+  "genesis": {
+    "hash": "0x0000000000000000000000000000000000000000000000000000000000000000",
+    "timestamp": "0x5bdda800",
+    "gasLimit": 10485760,
+    "difficulty": 1,
+    "nonce": "0x0000000000000000",
+    "extraData": "0x2249276d20646f6e652077616974696e672e2e2e20666f7220626c6f636b2066696e616c69747922202d2049676779270000000001fa1804c408085d9c57eeb167ce953c99b6cb1e20794Fd02933F303FbA550bd1fe2f0649E3576eB0000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000",
+    "stateRoot": ""
+  },
+  "hardforks": [
+    {
+      "name": "chainstart",
+      "block": 0,
+      "consensus": "poa",
```

Comment:
I think this should say `clique` instead.

Reply:
agree. we should also modify ``chains/rinkeby.json`` to use ``"consensus": "clique"``

Reply:
Yes, also agree. I would nevertheless like to for now keep within this PR and just put the goerli config into the existing format for early experimentation and make these kind of updates part of a second round of larger changes and generally rework our terminology here to at least come closer to a common genesis file format, see https://github.com/ethereumjs/ethereumjs-common/issues/30.